astronomer_starship/compat/starship_compatability.py

```diff
@@ -1134,6 +1161,164 @@ def dag_run_attrs(self):
         }
         return attrs
 
+    def task_log_attrs(cls) -> "Dict[str, AttrDesc]":
+        return {
+            "dag_id": {
+                "attr": "dag_id",
+                "methods": [
+                    ("GET", True),
+                    ("POST", True),
+                    ("DELETE", True),
+                ],
+                "test_value": "dag_0",
+            },
+            "run_id": {
+                "attr": "run_id",
+                "methods": [
+                    ("GET", True),
+                    ("POST", True),
+                    ("DELETE", True),
+                ],
+                "test_value": "manual__1970-01-01T00:00:00+00:00",
+            },
+            "task_id": {
+                "attr": "task_id",
+                "methods": [
+                    ("GET", True),
+                    ("POST", True),
+                    ("DELETE", True),
+                ],
+                "test_value": "task_id",
+            },
+            "map_index": {
+                "attr": "map_index",
+                "methods": [
+                    ("GET", True),
+                    ("POST", True),
+                    ("DELETE", True),
+                ],
+                "test_value": -1,
+            },
+            "try_number": {
+                "attr": "try_number",
+                "methods": [
+                    ("GET", True),
+                    ("POST", True),
+                    ("DELETE", True),
+                ],
+                "test_value": 0,
+            },
+            "log": {
+                "attr": "log",
+                "methods": [("POST", True)],
+                "test_value": "This is a test task log.",
+            },
+        }
+
+    @classmethod
+    def _task_log_path(
+        cls,
+        *,
+        dag_id,
+        run_id,
+        task_id,
+        map_index,
+        try_number,
+        **_,
+    ) -> tuple[str, str | None]:
+        """Get the path to the task log file and the connection ID for remote storage."""
+        ASTRONOMER_ENVIRONMENT = os.getenv("ASTRONOMER_ENVIRONMENT")
```

Comment:
default?
```suggestion
        ASTRONOMER_ENVIRONMENT = os.getenv("ASTRONOMER_ENVIRONMENT", "NOTSET")
```

Reply:
nvm

Reply:
Yeah, default is `None` which should result in a 409 complaining not being in an astronomer env.